### ISSUE TITLE
Confirm go-cose v1.0.0-rc.1 is ready for a 1.0.0 release

### ISSUE THREAD

**Issue by SteveLasker** [notaryproject/notation-core-go#37]
The go-cose project is pending a [1.0.0 release](https://github.com/veraison/go-cose/milestone/2)
Before cutting the release, they (me) are asking for dependent projects to confirm confidence in the current [1.0.0-rc.1](https://github.com/veraison/go-cose/releases/tag/v1.0.0-rc.1) release before cutting the 1.0.0 release.

Have we completed our testing for notary cose support? Once we have, we would add a +1 from notary on the [Create 1.0.0 Release #98](https://github.com/veraison/go-cose/issues/98) issue

**Comment by shizhMSFT** [comment]
@SteveLasker We are building and testing COSE support for notation, which can be tracked at
- notation-core-go: https://github.com/notaryproject/notation-core-go/pull/26
- notation-go: https://github.com/notaryproject/notation-go/pull/107
- notation: https://github.com/notaryproject/notation/pull/302

**Comment by SteveLasker** [comment]
@shizhMSFT, excellent! As we get cose integrated in a notation build, we'll start having users validate MAR content. At that point, we should have good validation. 
Or, do you think the test cases above have covered enough validation that you're confident in releasing go-cose 1.0?

**Comment by shizhMSFT** [comment]
@SteveLasker Let's release go-cose 1.0 after merging the COSE support for notation.

/cc @yizha1 @FeynmanZhou

**Comment by yizha1** [comment]
Once COSE support is merged into notation main branch, we could invite users to verify it for an E2E flow.  Based on the results we will be more confident on go-cose 1.0 release.

**Comment by shizhMSFT** [comment]
@SteveLasker Can we cut a `v1.0.0-rc.2` version for `go-cose`?

**Comment by FeynmanZhou** [comment]
go-cose v1.0.0-rc.2 is available: https://github.com/veraison/go-cose/releases/tag/v1.0.0-rc.2. I think we can start to update the dependency version of go-cose to v1.0.0-rc.2 [here](https://github.com/notaryproject/notation-core-go/blob/main/go.mod#L7).

**Comment by shizhMSFT** [comment]
I think `go-cose v1.0.0-rc.2` is good enough for a `go-cose v1.0.0` release to be consumed by notation.

**Comment by fanndu** [comment]
@SteveLasker The go-cose v1.1.0 is already supported, so this Issues should be closed.
`require (
	github.com/fxamacker/cbor/v2 v2.4.0
	github.com/golang-jwt/jwt/v4 v4.5.0
	github.com/veraison/go-cose v1.1.0
	golang.org/x/crypto v0.11.0
)`

**Comment by yizha1** [comment]
Thanks @fanndu Closed as https://github.com/veraison/go-cose/releases/tag/v1.1.0 was released
